src/install/kubernetes/guided/install-guided.mdx

```diff
@@ -0,0 +1,36 @@
+---
+componentType: default
+headingText: Install the Kubernetes integration
+---
+
+import kubernetesGuidedInstall from 'images/kubernetes_screenshot-crop_guided-install-ui.webp'
+
+import kubernetesGuidedInstallCommand from 'images/kubernetes_screenshot-crop_guided-install-ui-command.webp'
+
+
+
+The guided installation gives you control over the process, allowing you to choose which features to enable and which data to collect. It makes the installation process easier and simpler than the other ways to install the integration. It also offers a quickstart option that includes some optional, pre-built resources such as dashboards and alerts alongside the Kubernetes integration so that you can gain instant visibility into your Kubernetes clusters.
+
+From the guided install, you can choose from one of these instrumentation methods:
+
```

Comment:
I am confused about why you can choose `Helm` or `Manifest` for the guided install? I think this needs to be made a little more clear. How do those two elements fit _within_ the guided install where you only need to run a simple script?

Reply:
This is the UI where you can select one of these options. Each option is explained below. You can install it with Helm using the nri-bundle chart or with Manifest using the manifest yaml file.